test/circuit/library/ansatzes/test_ucc.py

```diff
@@ -133,88 +253,3 @@ def custom_excitations(num_spin_orbitals, num_particles):
-            ucc.num_particles = (1, 1)
-            self.assertEqual(ucc.num_particles, (1, 1))
-            self.assertIsNone(ucc.operators)
-            with self.assertRaises(ValueError):
-                _ = ucc.data
-
-        with self.subTest("Set num spin orbitals"):
-            ucc.num_spin_orbitals = 4
-            self.assertEqual(ucc.num_spin_orbitals, 4)
-            self.assertIsNone(ucc.operators)
-            with self.assertRaises(ValueError):
-                _ = ucc.data
-
-        with self.subTest("Set excitations"):
-            ucc.excitations = "sd"
-            self.assertEqual(ucc.excitations, "sd")
-            self.assertIsNone(ucc.operators)
-            with self.assertRaises(ValueError):
-                _ = ucc.data
-
-        with self.subTest("Set qubit converter to complete build"):
-            converter = QubitConverter(JordanWignerMapper())
-            ucc.qubit_converter = converter
-            self.assertEqual(ucc.qubit_converter, converter)
-            self.assertIsNotNone(ucc.operators)
-            self.assertEqual(len(ucc.operators), 3)
-            self.assertEqual(ucc.num_qubits, 4)
-            self.assertIsNotNone(ucc.data)
-
-        with self.subTest("Set custom operators"):
-            self.assertEqual(len(ucc.operators), 3)
-            ucc.operators = ucc.operators[:2]
-            self.assertEqual(len(ucc.operators), 2)
-            self.assertEqual(ucc.num_qubits, 4)
-
-        with self.subTest("Reset operators back to as per UCC"):
-            ucc.operators = None
-            self.assertEqual(ucc.num_qubits, 4)
-            self.assertIsNotNone(ucc.operators)
-            self.assertEqual(len(ucc.operators), 3)
-
-        with self.subTest("Set num particles to include 0"):
-            ucc.num_particles = (1, 0)
-            self.assertEqual(ucc.num_particles, (1, 0))
-            self.assertIsNotNone(ucc.operators)
-            self.assertEqual(len(ucc.operators), 1)
-
-        with self.subTest("Change num particles"):
-            ucc.num_particles = (1, 1)
-            self.assertIsNotNone(ucc.operators)
-            self.assertEqual(len(ucc.operators), 3)
-
-        with self.subTest("Change num spin orbitals"):
-            ucc.num_spin_orbitals = 6
-            self.assertIsNotNone(ucc.operators)
-            self.assertEqual(len(ucc.operators), 8)
-
-        with self.subTest("Change excitations"):
-            ucc.excitations = "s"
-            self.assertIsNotNone(ucc.operators)
-            self.assertEqual(len(ucc.operators), 4)
-
-        with self.subTest("Change qubit converter"):
-            ucc.qubit_converter = QubitConverter(ParityMapper(), two_qubit_reduction=True)
-            # Has not been used to convert so we need to force it to do two qubit reduction
-            ucc.qubit_converter.force_match(ucc.num_particles)
-            self.assertIsNotNone(ucc.operators)
-            self.assertEqual(ucc.num_qubits, 4)
```

Comment:
I don't think these tests were planned to be removed. Looks like a bad merge since these were only added back in March